Apps/Database/src/Delegates/DBCommunicationDelegate.cs

```diff
@@ -55,7 +55,10 @@ public DBResult<Communication> GetActiveBanner()
             };
             result.Payload = this.dbContext.Communication
                 .OrderByDescending(c => c.CreatedDateTime)
-                .FirstOrDefault(c => c.CommunicationTypeCode == CommunicationType.Banner && (DateTime.UtcNow >= c.EffectiveDateTime && DateTime.UtcNow <= c.ExpiryDateTime));
+                .Where(c => c.CommunicationTypeCode == CommunicationType.Banner)
+                .Where(c => c.CommunicationStatusCode != CommunicationStatus.Draft)
```

Comment:
Since there are other statuses I think == CommunicationStatus.New would be better